spinalcordtoolbox/reports/qc.py

```diff
@@ -296,8 +318,10 @@ def update_description_file(self, dimension):
         # get path of the toolbox
 
         output = {
+            'python': sys.executable,
+            'cwd': self.qc_params.cwd,
+            'cmdline': "{} {}".format(self.qc_params.command, self.qc_params.args),
```

Comment:
Consider using this formatting style on the other strings also. Sometimes you're using the `"%s" % str` and sometimes `.format()`.

Reply:
Most of the times I use `str.format()` but sometimes I use the other; IMHO both are useful and have preferred contexts of application.

Here `str.format()` is arguably better because ("%s %s" % (a, b)) would have involved an extra parenthesis, and broken with the surrounding lines.

Reply:
The old-style formatting will be deprecated in future, it is still there for backwards compatibility. So if we want to be careful with Python 3 and further we'd better get rid of these old string formatting. They also added a message [regarding this](https://docs.python.org/2/library/stdtypes.html#str.format) in the Python 2.x docs:

    This method of string formatting is the new standard in Python 3,
    and should be preferred to the % formatting described in String
    Formatting Operations in new code.